### ISSUE TITLE
Visualisations: Mosaic plot updates to axis labels

### ISSUE THREAD

**Issue by dmfalke** [VEuPathDB/web-components#151]
1. Remove “(%)” from the end of both the x-axis and y-axis labels
2. Remove the proportion numbers from the x-axis, and replace with the values for the x-axis variable
3. Put “Proportion” in front of (or on top of) the y-axis label
4. Add the y-axis variable label to the legend
5. _(Possibly remove the counts and proportions next to the x-axis values)_

![image](https://user-images.githubusercontent.com/365139/121952463-40896a00-cd2a-11eb-8d6b-ca3431a0f70e.png)

**Comment by bobular** [comment]
Please see #150 - merge in first before starting this!

**Comment by aaronwlsong** [comment]
QA checking:

**QA 1. For the same study and the same value of x-axis and y-axis:** 

_1.Remove “(%)” from the end of both the x-axis and y-axis labels_
Not change on QA website, the (%)is still after y-axis label;

_2. Remove the proportion numbers from the x-axis, and replace with the values for the x-axis variable_
Not change on QA website, the proportion number still shows on x-axis;

_3. Put “Proportion” in front of (or on top of) the y-axis label_
Not change on website, there is no proportion on top /in front of the y-axis label;

_4.Add the y-axis variable label to the legend_
Not change on website, there is no legend;

_5. (Possibly remove the counts and proportions next to the x-axis values)_
Not found the counts next to the x-axis values

<img width="1546" alt="WeChat4414ef365a11056b0c63fddb034491c9" src="https://user-images.githubusercontent.com/61077845/130079489-1e7455ef-0a41-4142-9148-30af96b3f095.png">

**QA 2. For the same study and different value of y-axis:** 

_1.Remove “(%)” from the end of both the x-axis and y-axis labels_
Changed on QA website, the (proportion) is after y-axis label in stead of (%), but not for x-axis;

_2. Remove the proportion numbers from the x-axis, and replace with the values for the x-axis variable_
Changed on QA website, the value of x-axis shows on x-axis;

_3. Put “Proportion” in front of (or on top of) the y-axis label_
Change on QA website, there is (proportion) after the y-axis label;

_4.Add the y-axis variable label to the legend_
Changed on QA website, there is a legend;

_5. (Possibly remove the counts and proportions next to the x-axis values)_
Not changed.

<img width="1610" alt="WeChat7e38395747eec888d66c82a937330ed2" src="https://user-images.githubusercontent.com/61077845/130080718-3b360f2f-1e6b-4ccb-b03c-3ace67b60a67.png">

**QA.3 Other issues for the same study with different values of y-axis.**
User is able to choose variable for y-axis, but it could not generate figure.

<img width="2014" alt="WeChatf96a352994bccdfb8ad97f0ce81c138b" src="https://user-images.githubusercontent.com/61077845/130081250-63fe273a-696f-47eb-9078-957bebe80c4c.png">
 
**QA 4. For the same study and different value of x-axis but the same y-axis:** 

_1.Remove “(%)” from the end of both the x-axis and y-axis labels_
Changed on QA website, the (proportion) is after y-axis label in stead of (%), but not for x-axis;

_2. Remove the proportion numbers from the x-axis, and replace with the values for the x-axis variable_
Changed on QA website, the value of x-axis shows on x-axis;

_3. Put “Proportion” in front of (or on top of) the y-axis label_
Change on QA website, there is (proportion) **after** the y-axis label;

_4.Add the y-axis variable label to the legend_
Changed on QA website, there is a legend;

_5. (Possibly remove the counts and proportions next to the x-axis values)_
Not changed.

![image](https://user-images.githubusercontent.com/61077845/130082220-92b0dd4d-187e-43eb-a3aa-0cfad3e39042.png)

**QA 5. For the same study and different value of x-axis and y-axis:** 
Same as QA.4 and QA.3
![image](https://user-images.githubusercontent.com/61077845/130082471-8b041ebd-9a4d-44ec-ac58-d956533fd035.png)

<img width="1600" alt="WeChata8ac6354361acf6bca6cd7cb89e55a11" src="https://user-images.githubusercontent.com/61077845/130082618-44a3218e-217c-48e3-b094-4270882648e2.png">

**Comment by danicahelb** [comment]
@dmfalke changes are not showing up on qa.clinepidb.org

**Comment by danicahelb** [comment]
@aaronwlsong when you encounter a new bug or other issue when performing QA, please open a new ticket instead of adding to the original ticket

**Comment by danicahelb** [comment]
@chowington assigning back to you, issues haven't been addressed

**Comment by chowington** [comment]
The latter two images look correct to me; I don't know why the first would be incorrect. Maybe the site was accessed before being updated for the first QA case. Anyway, it looks correct to me on QA now. Pushing back to ready for review.

**Comment by SheenaTomko** [comment]
Looks good
